### ISSUE TITLE
$subtext variable has insufficient contrast per WCAG 2.1

### ISSUE THREAD

**Issue by reidelliott** [woocommerce/woocommerce#24610]
**Is your feature request related to a problem? Please describe.**
The `$subtext` SCSS variable, which is used on `.breadcrumb` is `#777777`, which, if on white, renders color contrast errors.

> ... has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.48:1. Recommendation: change text colour to #767676.

https://github.com/woocommerce/woocommerce/issues/24185#issuecomment-530186565

**Describe the solution you'd like**
Change variable's value to `#767676`.

**Describe alternatives you've considered**
The alternative requires overriding in the theme.

**Comment by claudiosanches** [comment]
Thank you for reporting this issue!

I'm closing the issue as it already has a PR, please continue the discussion over at #24739.
